lib/util/time.ex

```diff
@@ -154,11 +153,11 @@ defmodule Antikythera.Time do
   end
 
   defun to_epoch_milliseconds(t :: v[t]) :: integer do
-    to_gregorian_milliseconds(t) - @time_epoch_offset_milliseconds
+    to_gregorian_milliseconds(t) - MilliSecondsInGregorian.time_epoch_offset_milliseconds
```

Comment:
Make sure to add parentheses when calling a function.
https://hexdocs.pm/antikythera/style_guide.html#function-calling-syntax

The same applies to L.160.

Reply:
Thank you for telling me a style guide.

Reply:
82e73a51b30711f73e305bc31acdedfc202c9181